### PR TITLE
Species has custom brute/burn/cloning damage examine

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Different human species can now have different examine texts, allowing androids to not have bruising, and instead have denting, like a cyborg would.

## Why It's Good For The Game

It's a little more customization to some species who don't exactly have flesh and bone to be bruised and burned normally.

## Changelog

:cl:
add: Species now have custom damage examines, so Androids now have denting and charring, rather than bruising and burning. 
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
Could you put these on the limbs instead? That way someone who has augments will also get dents instead of bruises

**Comment by necromanceranne** [comment]
I thought cybernetic limbs had this already actually.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -179,29 +179,29 @@
...
+				msg += "<B>[t_He] [t_has] severe [dna.species.brute_damage_desc]!</B>\n"
 
 		temp = getFireLoss()
 		if(temp)
 			if(temp < 25)
-				msg += "[t_He] [t_has] minor burns.\n"
+				msg += "[t_He] [t_has] minor [dna.species.burn_damage_desc].\n"
 			else if (temp < 50)
-				msg += "[t_He] [t_has] <b>moderate</b> burns!\n"
+				msg += "[t_He] [t_has] <b>moderate</b> [dna.species.burn_damage_desc]!\n"
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/examine.dm:193]: this doesn't runtime on dummies right?

**Comment by chemistrymain2** [comment]
skeletons should have chipping and charring too

File: code/__DEFINES/mobs.dm [hunk 2]
```diff
@@ -440,6 +440,13 @@
 #define MAX_REVIVE_FIRE_DAMAGE 180
 #define MAX_REVIVE_BRUTE_DAMAGE 180
 
+#define DEFAULT_BRUTE_EXAMINE_TEXT "bruising"
+#define DEFAULT_BURN_EXAMINE_TEXT "burns"
+#define DEFAULT_CLONE_EXAMINE_TEXT "cellular damage"
+
+#define ROBOTIC_BRUTE_EXAMINE_TEXT "denting"
+#define ROBOTIC_BURN_EXAMINE_TEXT "charrings"
```
Comment (dragomagol) [inline code/__DEFINES/mobs.dm:448]: Would probably just say "charring"

Comment (dragomagol) [inline code/__DEFINES/mobs.dm:448]:
```suggestion
#define ROBOTIC_BURN_EXAMINE_TEXT "charring"
```

**Comment by dragomagol** [comment]
Agree that it would be nice for other species to have custom damages, but that can be done in another PR

File: code/__DEFINES/mobs.dm (hunk 2 above)
Comment (JohnFulpWillard) [reply code/__DEFINES/mobs.dm:448]: It's done to be consistent with burns, 

``He has moderate charrings.`` vs ``He has moderate charring.``

**Pull request merged**